circle.yml

```diff
@@ -1,3 +1,7 @@
+machine:
+  environment:
+    RECAPTCHA_PUBLIC_KEY: 6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI
```

Comment:
sorry, still not the place for them. The goal is to keep all values of this sort out of the source control.
CircleCI has an interface to set env vars similar to heroku's.